crates/astria-cli/README.md

```diff
@@ -0,0 +1,70 @@
+# Astria CLI
+
+## Dependencies
+
+- kubectl
+
+## Building
+
+```sh
+cargo build --release
+```
+
+## Running
+
+```sh
+# from monorepo root
+./target/release/astria-cli --help
+
+# examples:
+
+# create a rollup config
+./target/release/astria-cli rollup config create \
+  --use-tty \
+  --log-level DEBUG \
```

Comment:
it seems strange to me that the log level is set here - is this basically just passed into the config of the rollup (conductor)?

Reply:
This gets passed to any astria components, so Conductor and Composer in this scenario. I think it makes sense to let a user who is deploying their own rollup to set their desired log level.